.travis.yml

```diff
@@ -66,8 +66,7 @@ script:
  - if [[ -z $CIBW_BEFORE_BUILD ]]; then py.test -v --timeout 20 --ignore=tmp-build --import-mode append ; fi
  - if [[ -n $TRAVIS_TAG && -n $CIBW_BEFORE_BUILD ]]; then cibuildwheel --output-dir wheelhouse1 && tools/fixup-wheels.sh wheelhouse1 wheelhouse ; fi
  - if [[ -n $TRAVIS_TAG && $TRAVIS_OS_NAME == linux && -n $CIBW_BEFORE_BUILD ]]; then tools/test-manylinux.sh ; fi
- - if [[ -n $TRAVIS_TAG && -n $CIBW_BEFORE_BUILD ]]; then pip install --no-index -f wheelhouse/ confluent-kafka;\
-		 py.test -v --timeout 20 --ignore=tmp-build --import-mode append; tests/test_misc.py::test_unordered_dict; fi 
+ - if [[ -n $TRAVIS_TAG && -n $CIBW_BEFORE_BUILD ]]; then pip install --no-index -f wheelhouse/ confluent-kafka && cd .. && pytest -v --timeout 20 --ignore=tmp-build --import-mode append confluent-kafka-python/tests/test_misc.py::test_unordered_dict; fi
```

Comment:
Put the cd and the command in a subshell, or cd .. back out, otherwise any sub-sequent commands will fail because the directory changed:

` - if [[ -n $TRAVIS_TAG && -n $CIBW_BEFORE_BUILD ]]; then pip install --no-index -f wheelhouse/ confluent-kafka && (cd .. && pytest -v --timeout 20 --ignore=tmp-build --import-mode append confluent-kafka-python/tests/test_misc.py::test_unordered_dict); fi`

Reply:
True, I guess there are no guarantees this will always be the last script executed

Reply:
Right, it will help the next poor chap that adds a command here.